www/views/page/content/list.php

```diff
@@ -76,7 +99,10 @@ function cap($text, $len)
     if (!empty($catFilter)) {
         $filtered = $filtered || !(in_array($article->category_id, $catFilter));
     }
-    if (isset($filter)) {
+    if ($role == 1 && $filtercat != 0) {
```

Comment:
PHPStan klaagt hier dat je niet zeker weet of `$filtercat` is ingesteld. Dat is wel zo als `$role == 1` maar gebeurt zo ver terug dat het denk ik beter is om te zorgen dat `$filtercat` gewoon altijd 0 is als het niet iets anders is.

Vergeet ook niet altijd even `php phpstan.phar` te draaien.